http/mediatypes/mediatypes.go

```diff
@@ -0,0 +1,7 @@
+package mediatypes
+
+// Media types
+const (
+	ApplicationJSON     = "application/json"
+	ApplicationJSONUtf8 = "application/json; charset=utf-8"
```

Comment:
nit: Go style mandates that initialisms are always the same case. Here:

ApplicationJSONUTF8

Reply:
outdated: I've decided to start with `application/json`